x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/legacy_url_conflict_callout.tsx

```diff
@@ -27,7 +27,7 @@ export const LegacyUrlConflictCallOut = React.memo<LegacyUrlConflictCallOutProps
           <EuiSpacer />
           {spacesApi.ui.components.getLegacyUrlConflict({
             objectNoun: i18nTranslate.translate(
-              'xpack.triggersActionsUI.sections.ruleDetails.redirectObjectNoun',
+              'xpack.securitySolution.triggersActionsUI.sections.ruleDetails.redirectObjectNoun',
```

Comment:
Anything inside `x-pack/plugins/security_solution` should be prefixed by `xpack.securitySolution.*`